packages/server/test/profile.integration.ts

```diff
@@ -14,6 +14,107 @@ import { DesktopNotifModel } from 'notification/desktop-notif.entity';
 describe('Profile Integration', () => {
   const supertest = setupIntegrationTest(ProfileModule);
 
+  describe('POST /profile/:id/edit_user', () => {
+    it('Does not allow unauthorized edit', async () => {
```

Comment:
does this check if other user attempts to change other user's details? Perhaps make it more descriptive?